content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -11,7 +11,7 @@ toc: false
 
 To connect your database, you need to set the `url` field of the `datasource` block in your Prisma schema to your database [connection URL](/reference/database-reference/connection-urls):
 
-<SwitchTech technologies={['*', 'postgresql']}>
+<SwitchTech technologies={['*', 'postgres']}>
```

Comment:
The correct terminology is `postgresql` in Prisma land. I would prefer if we could use that everywhere instead of `postgres` to be consistent.

Reply:
I wanted to change it to `postgresql` as well, but that would involve a lot of redirects 😐

Reply:
Uh interesting, we had that online all that time really?

Google indeed knows ~30 pages: https://www.google.com/search?q=site:prisma.io+inurl:www+inurl:docs+inurl:postgres&rlz=1C1GCEA_enDE994DE994&filter=0&biw=2560&bih=1262&dpr=1.5

Reply:
@janpio So what do you think? Go with `postgres` for now or make it `postgresql` with redirects?

Reply:
I think `postgresql` would be the cleaner and more consistent solution.